pkg/sm/plugins/noop/noop.go

```diff
@@ -33,22 +33,22 @@ func (u *plugin) Spec() string {
 }
 
 func (p *plugin) Validate() error {
-	glog.V(3).Info("noop Plugin Validate()")
+	log.Info().Msg("noop Plugin Validate()")
```

Comment:
imo noop plugin should have all prints as debug WDYT?

Reply:
I think we should keep it as Info it just example to the developer.